app/Http/Controllers/UserController.php

```diff
@@ -23,16 +23,18 @@ public function store(Request $request)
         $request->validate([
             'first_name' => 'required',
             'last_name' => 'required',
-            'email' => 'required',
+            'email' => 'required|unique:users',
             'password' => 'required',
             'description' => '',
         ]);
 
+        if($request['description'] == null)
```

Comment:
```suggestion
        if($request['description'] === null)
```
PHP is one of those language that need the triple equals to really work